src/containers/shared/utils.js

```diff
@@ -151,7 +151,10 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
   return formatted.split('1')[0].trim();
 };
 
-export const formatLargeNumber = (d = 0, digits = 4) => {
+export const formatLargeNumber = (d = 0, digits = 6) => {
```

Comment:
Why did the default number of digits change? You can just pass in a different value for the variable and then you have to worry less about differences

Reply:
Either way is fine!